app/controllers/developer_controller.rb

```diff
@@ -13,7 +13,14 @@ def update_sites
 
   def production_log
     @log = if params[:grep].present?
-             `tail -n 10000 log/production.log | grep -E '#{params[:grep]}' -C #{params[:context]} --color=never`
+             grep_str = params[:grep].to_s
```

Comment:
This is very minor, but I'd personally prefer to inline this (`params[:grep].to_s.tr("'", '')`)